react/react1/week2/todo-app/src/App.js

```diff
@@ -1,22 +1,24 @@
-function App() {
+import styled from 'styled-components';
+import Timer from './components/Timer';
+import Title from './components/Title';
+import TodoList from './components/TodoList';
+
+const Wrapper = styled.div`
+  align-items: center;
+  background: #d5ffe3;
+  display: flex;
+  flex-direction: column;
+  font-size: 2rem;
+`;
+
+const App = () => {
```

Comment:
Your code very clean as always, its really good to have a separate components, but I still have a difficulty to call the component 😅